README.md

```diff
@@ -83,13 +52,8 @@ Flags:
       --cluster-name string                  The name of target cluster (required)
       --cluster-namespace string             The namespace of target cluster (required)
   -h, --help                                 help for ./bin/cluster-api-upgrade-tool
-      --image-field string                   The image identifier field in provider manifests (optional)
-      --image-id string                      The provider-specific image identifier to use when booting a machine (optional)
       --kubeconfig string                    The kubeconfig path for the management cluster
       --kubernetes-version string            Desired kubernetes version to upgrade to (required)
-      --machine-deployment-name string       Name of a single machine deployment to upgrade
-      --machine-deployment-selector string   Label selector used to find machine deployments to upgrade
-      --scope string                         Scope of upgrade - [control-plane | machine-deployment] (required)
       --upgrade-id string                    Unique identifier used to resume a partial upgrade (optional)
```

Comment:
can be out of scope for this PR if you like, but this is required. If left empty I get a stack trace:

```
root@ip-172-31-60-121:~/cluster-api-upgrade-tool# ./cluster-api-upgrade-tool --cluster-name my-cluster --cluster-namespace default --kubeconfig /etc/kubernetes/admin.conf --kubernetes-version v1.16.2
Error: upgrade ID must be a timestamp containing only digits
upgrade ID must be a timestamp containing only digits
github.com/vmware/cluster-api-upgrade-tool/pkg/upgrade.NewControlPlaneUpgrader
	/home/ubuntu/cluster-api-upgrade-tool/pkg/upgrade/control_plane.go:75
main.main.func1
	/home/ubuntu/cluster-api-upgrade-tool/main.go:31
github.com/spf13/cobra.(*Command).execute
	/home/ubuntu/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:762
github.com/spf13/cobra.(*Command).ExecuteC
	/home/ubuntu/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:852
github.com/spf13/cobra.(*Command).Execute
	/home/ubuntu/go/pkg/mod/github.com/spf13/cobra@v0.0.3/command.go:800
main.main
	/home/ubuntu/cluster-api-upgrade-tool/main.go:89
runtime.main
	/home/ubuntu/.gimme/versions/go1.12.12.linux.amd64/src/runtime/proc.go:200
runtime.goexit
	/home/ubuntu/.gimme/versions/go1.12.12.linux.amd64/src/runtime/asm_amd64.s:1337
```

Reply:
Let's fix it separately

Reply:
for sure https://github.com/vmware/cluster-api-upgrade-tool/issues/108